robot/EDA/resources/EDA.py

```diff
@@ -394,3 +394,17 @@ def verify_selected_dropdown_value(self,**kwargs):
             actual_value = self.selenium.get_webelement(locator).text
             if not str(value).lower() == str(actual_value).lower() :
                 raise Exception (f"Drop down value in '{field}' is not updated and the value is '{actual_value}'")
+
+    def verify_dropdown_field_status(self, **kwargs):
+        """ Verify the drop down field is disabled/enabled for the user
+            we have to pass the name of the field and the status of the field 
+            True = disabled and False = enabled
+        """
+        for field,expected_value in kwargs.items():
+            locator = eda_lex_locators["eda_settings"]["dropdown_field"].format(field)
+            self.selenium.page_should_contain_element(locator)
+            self.selenium.wait_until_element_is_visible(locator,
+                                                error= f"Element '{field}' is not displayed for the user")
+            actual_value = self.selenium.get_webelement(locator).get_attribute("disabled")
+            if not str(expected_value).lower() == str(actual_value).lower() :
+                raise Exception (f"Drop down field '{field}' status is '{actual_value}'")
```

Comment:
just a suggestion, may be little more detailed like Expected '{field}' Dropdown status to be {expected_value} bout found '{actual_value}

Reply:
also u dont need quotes around {} again in f string

Reply:
Updated both